tests/test_notifications.py

```diff
@@ -262,7 +261,7 @@ def test_node_subscriptions_and_backrefs_removed_when_node_is_deleted(self):
             subscription.save()
 
             s = getattr(project.creator, 'email_transactional', [])
-            assert_equal(len(s), 1)
+            assert_equal(len(s), 2)
```

Comment:
Looks like it's more complex; I find a `__backrefs` key with an `email_transactional` key. I guess these `__backref`s are exposed as attributes on the `AuthUserFactory`s?

![screen shot 2016-02-23 at 1 50 14 pm](https://cloud.githubusercontent.com/assets/134455/13262757/7b7a8886-da34-11e5-9152-e353f5ee8c4e.png)